src/mockdata/mocks.ts

```diff
@@ -4,6 +4,7 @@ import { WEATHER_MET_API } from '../queries/metApi';
 import { METADATA, MetadataEntry } from '../queries/metadata';
 import faker from 'faker';
 import { DataSourceName } from '../utils/dataSourceMappings';
+import { VisualisationType } from '../types/Metadata';
 
 faker.seed(42069);
```

Comment:
I don't really appreciate these kinds of connotations.

Reply:
Ok/
Same